entity-service/src/main/kotlin/com/egm/stellio/entity/service/EntityOperationService.kt

```diff
@@ -150,28 +151,30 @@ class EntityOperationService(
      */
     @Transactional(rollbackFor = [BadRequestDataException::class])
     @Throws(BadRequestDataException::class)
-    fun replaceEntity(entity: NgsiLdEntity): Either<BatchEntityError, URI> {
+    fun replaceEntity(entity: NgsiLdEntity): Either<BatchEntityError, BatchEntitySuccess> {
         neo4jRepository.deleteEntityAttributes(entity.id)
         val (_, notUpdated) = entityService.appendEntityAttributes(entity.id, entity.attributes, false)
-        if (notUpdated.isEmpty()) {
-            return entity.id.right()
-        } else {
+        if (notUpdated.isEmpty())
+            return BatchEntitySuccess(entity.id).right()
+        else
             throw BadRequestDataException(
                 ArrayList(notUpdated.map { it.attributeName + " : " + it.reason }).joinToString()
```

Comment:
Is it supposed to be returned in expanded format ?

Reply:
The specification mention that attributes should be represented by their names.
So I'm not sure about the format.

Reply:
not part of the PR, leave them as they are currently.